scout/server/blueprints/cases/templates/cases/gene_panel.html

```diff
@@ -7,6 +7,7 @@
         <thead class="table-light thead">
           <tr style="cursor: pointer; white-space: nowrap">
             <th>Panel <i class="fas fa-sort" data-bs-toggle="tooltip" title="Sort by gene panel name"></i></th>
+            <th><i class="fas fa-bars float-end mx-4" data-bs-toggle="tooltip" title="Sort by default panel"></i></th>
```

Comment:
```suggestion
            <th><i class="fas fa-sort float-end mx-4" data-bs-toggle="tooltip" title="Sort by default panel"></i></th>
```

Reply:
Works fine, but perhaps use fa-sort to make is similar to the other columns?

Reply:
![image](https://github.com/Clinical-Genomics/scout/assets/112548186/8e0467e3-365e-4538-a991-ccc684428927)

Updated icon and added a header as well as removed some very specific styling of the default column